### ISSUE TITLE
Proposal to Translate Website into Haitian Creole and French

### ISSUE THREAD

**Issue by Jim707t** [dair-ai/Prompt-Engineering-Guide#75]
I am interested in contributing to this project by translating the website into Haitian Creole and French. This will make the content accessible to a wider audience and help promote rapid engineering knowledge sharing.

Being bilingual, I can therefore effectively translate technical and educational content.
Thank you for considering my proposal.

**Comment by omarsar** [comment]
@Jim707t Thanks for your interest in helping with translation. To translate the content into a language, say, French, you can simply create a `fr-pages` folder in the root. You can then translate the `<>.en.mdx` files found in the `pages` folder and make sure to keep the same folder structure in your `fr-pages` folder. In other words, you don't touch the `pages` folder since your translated files will all be in the `fr-pages` folder. Once you finish with the translations, you can submit PR and I will take care of integrating it into the website. Does this work for you?

I suggest to also use a grammar tool that works for your language to make sure grammar is consistent and avoiding spelling errors in your translations. Looking forward to your contributions and thanks a lot.

**Comment by Jim707t** [comment]
> 

Thanks for the additional advice. I will update you on my progress and submit the PR once I'm finished translating the content into Haitian Creole and French.

**Comment by silopolis** [comment]
@Jim707t hey, dumbly, I didn't check for relevant issues before starting! :/ I'm too willing to contribute French translation, so maybe we can share the work with Haitian Creole, WDYT?

I have started setting up a Crowdin project for that here https://crowdin.com/project/prompt-engineering-guide

@omarsar that tip about putting the files into a dedicated tree is good news because I was struggling in Crowdin with Nextra's `%file_name%.%two_letter_code%.%file_extension%` pattern including for English source files. But let me open another issue about that…
